src/main/java/pl/kwolska/playground/adapter/api/TransferController.java

```diff
@@ -1,26 +1,50 @@
 package pl.kwolska.playground.adapter.api;
 
 import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
+import pl.kwolska.playground.domain.model.Account;
 import pl.kwolska.playground.domain.model.Transfer;
 import pl.kwolska.playground.domain.TransferService;
 
-import java.math.BigDecimal;
+import javax.persistence.EntityNotFoundException;
 import java.util.List;
 
 @RestController
 @AllArgsConstructor
 public class TransferController {
   
   private final TransferService transferService;
-  
-  @RequestMapping(value = "/account/{creditAccountId}/transfers", method = RequestMethod.POST)
-  public void addUserTransfer(@PathVariable int creditAccountId, Integer debitAccountId, BigDecimal money) {
-    transferService.createTransfer(debitAccountId, creditAccountId, money);
+
+  // todo 5: tu powinien @RequestBody (idki, money)
+  @RequestMapping(value = "/accounts/{creditAccountId}/transfers", method = RequestMethod.POST)
+  public void addUserTransfer(@PathVariable int creditAccountId, @RequestBody TransferDto transferDto) {
+    transferService.createTransfer(transferDto.getDebitAccountId(), creditAccountId, transferDto.getMoney());
   }
   
-  @RequestMapping(value = "/account/{accountId}/transfers", method = RequestMethod.GET)
+  @RequestMapping(value = "/createAccounts", method = RequestMethod.POST)
+  public void createAccounts() {
+    transferService.createAccounts();
+  }
+
+  // todo 4: /accounts/{id} - id, balance, transfery
+  @RequestMapping(value = "/accounts/{accountId}", method = RequestMethod.GET)
+  public ResponseEntity<AccountDto> findAccountById(@PathVariable Integer accountId) {
+    return transferService.findAccountById(accountId)
+        .map(account -> new AccountDto(account.getId(), account.calculateBalance(), account.getTransfers()))
+        .map(accountDto -> new ResponseEntity<>(accountDto, HttpStatus.OK))
+        .orElse(new ResponseEntity<>(HttpStatus.NOT_FOUND));
+  }
+
+  @RequestMapping(value = "/accounts/{accountId}/transfers", method = RequestMethod.GET)
```

Comment:
Jest pewna heurystyka jeżeli chodzi o tworzenie Restowych Controllerów w Springu i mapowanie URLi na nie.

Jak mamy klienta to zasoby zwiazane z nim udostepniamy pod `ClientController` i wszystkie endpointy w nim będą się zaczynały od `client`,
np. 
`GET /clients` - pobranie klientow
`POST /clients` - stworzenie nowego klienta
`GET /clients/1` - pobranie klienta o id 1
`GET /clients/1/orders` - pobranie zamówień dla klienta 1
`POST clients/1/orders` - stworzenie nowego zamowienia dla klienta

ale też możemy mieć np jakiś OrdersController
`GET /orders`
i być może
`POST /orders` stworzenie nowego zamowienia, ciezko powiedziec jak powinno byc jak nie znamy kontekstu...

Tutaj natomiast pasuje by transfery byly oddzielnie jako `POST /transfers` i zeby tez accounts tez mialy swoj kontroller.

Inaczej istnieje ryzyko ze zaczniemy wprowadzac po trochu i po woli bałagan i ugryźie nas to kiedyś w dupę.

Wiem, bo sam to robiłem i doświadczyłem na właśnej skórze. (i dupie)